ApiBundle/Controller/TranslateController.php

```diff
@@ -26,10 +26,14 @@ class TranslateController extends BaseController
     public function translateAction(Request $request)
     {
         $displayedElements = $request->get('displayedElements');
+
+        $entityType = $this->container->camelize($request->get('entityType'));
+        $entityType = $this->container->underscore($entityType);
```

Comment:
All right.

With such a logic, I would prefer to put those line of code in a transformer, this would be easier to write a test for all cases :)